### PR TITLE
Issue6 and 7

### PR DESCRIPTION
**Pull request by MarkLoeser**
Not sure if I did this correctly...
Contains both updated sponsors section and get involved accordion

### CODE REVIEW

File: views/index-getInvolved.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <div class="panel-heading" role="tab" id="headingThree" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="true" aria-controls="collapseThree">
+      <h4 class="panel-title">
+        <a class="collapsed" role="button">
+          Start or Contribute to a Project
+        </a>
+      </h4>
+    </div>
+    <div id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree">
+      <div class="panel-body">
+        <div class="col-sm-2 col-sm-offset-0 col-xs-4 col-xs-offset-4 involvedImage"><a href="#" target="_blank"><img src="/images/get_involved/github.png"></a></div>
```
Comment (TheJaredWilcurt) [inline views/index-getInvolved.html]: This image should link to our Campers page and not have an `target` attribute.

File: views/index-getInvolved.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    <div class="panel-heading" role="tab" id="headingFour" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
+      <h4 class="panel-title">
+        <a class="collapsed" role="button">
+          Add your Blog
+        </a>
+      </h4>
+    </div>
+    <div id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour">
+      <div class="panel-body">
+        <div class="col-sm-2 col-sm-offset-0 col-xs-4 col-xs-offset-4 involvedImage"><a href="#" target="_blank"><img src="http://res.cloudinary.com/dpyptkefp/image/upload/c_scale,w_120/v1476713784/fb_icon_325x325_r17eed.png"></a></div>
```
Comment (TheJaredWilcurt) [inline views/index-getInvolved.html]: `images/get_involved/facebook.png`

`https://www.facebook.com/groups/free.code.camp.indianapolis.in`

File: views/index-getInvolved.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    <div class="panel-heading" role="tab" id="headingFive" data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
+      <h4 class="panel-title">
+        <a class="collapsed" role="button">
+          Speak at a Meetup
+        </a>
+      </h4>
+    </div>
+    <div id="collapseFive" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFive">
+      <div class="panel-body">
+        <div class="col-sm-2 col-sm-offset-0 col-xs-4 col-xs-offset-4 involvedImage"><a href="#" target="_blank"><img src="/images/get_involved/meetup.png"></a></div>
```
Comment (TheJaredWilcurt) [inline views/index-getInvolved.html]: `alt` text and URL

**Comment by mattattaq** [comment]
ejwnxsenwyamambd-bfhjqsqptoaaksosizafvxdvbnxfdvnnlwrq6tphsdvgxkkpyvhkfkx5tjy6hxqyccekrcpm18yab23zjw36xd_19ictdqyubg51dxfnpc6perl6yh63yv6_qfxpin-

This would probably be a better way to show the sponsors. Also after looking at your branch I think the limited width version of the accordion flows a bit better.

**Comment by MarkLoeser** [comment]
I've made these changes and pushed them to my branch. I tried to pull down the new version and merge it with mine before pushing back to github but I kept getting merge conflicts with the style.css file with the long "@import......." line and wasn't sure how to fix that.

**Pull request merged**